mk/docker.mk

```diff
@@ -3,11 +3,18 @@ KUMA_VERSION ?= master
 
 DOCKER_SERVER ?= docker.io
 DOCKER_REGISTRY ?= $(DOCKER_SERVER)/kumahq
+DOCKER_DEV_IMAGE_SUFFIX ?=
 DOCKER_USERNAME ?=
 DOCKER_API_KEY ?=
 
+ifdef IMAGES_REPOSITORY_SUFFIX
+ifneq ($(strip $(IMAGES_REPOSITORY_SUFFIX)),)
+DOCKER_DEV_IMAGE_SUFFIX = -$(IMAGES_REPOSITORY_SUFFIX)
+endif
+endif
```

Comment:
Why the indirection? Feels like we could use `IMAGE_REPOSITORY_SUFFIX` directly in the script no?